www/src/components/feedback-widget/feedback-form.js

```diff
@@ -71,6 +71,9 @@ const textareaStyles = {
   height: `5.5rem`,
   mt: 1,
   mb: 4,
+  px: 3,
+  py: 2,
+  lineHeight: dense,
```

Comment:
```suggestion
  lineHeight: `dense`,
```
Should it be like this?

Reply:
Yep (`dense` without the backticks would be a variable).
I'll check out on localhost now!

Reply:
Got sidetracked, sorry! Will do what I promised earlier, and properly review now … expect this to be merged in ~10 minutes.